arangod/V8Server/v8-voccursor.cpp

```diff
@@ -339,6 +343,10 @@ struct V8Cursor final {
     
     // specify ID 0 so it uses the external V8 context
     auto cc = cursors->createQueryStream(std::move(q), batchSize, ttl);
+    if (cc == nullptr) {   // soft shutdown, no longer allowed
+      TRI_V8_THROW_EXCEPTION_FULL(TRI_ERROR_SHUTTING_DOWN,
+           "coordinator soft shutdown, new cursors blocked");
+    }
```

Comment:
As mentioned somewhere above, I would remove the check here and make `CursorRepository::createQueryStream()` throw an appropriate exception when we are in soft shutdown.

Reply:
Will do.

Reply:
Done.